modules/ROOT/pages/bolt/server-state.adoc

```diff
@@ -119,6 +119,14 @@ The `READY` state can handle the request messages `RUN` and `BEGIN` and receive
 ** `RUN` to `STREAMING` or `FAILED`
 ** `BEGIN` to `TX_READY` or `FAILED` (v3+)
 
+In v5.1, the following transitions are possible:
+
+** `<INTERRUPT>` to `INTERRUPTED`
+** `<DISCONNECT>` to `DEFUNCT`
+** `RUN` to `STREAMING` or `FAILED`
+** `BEGIN` to `TX_READY` or `FAILED`
+** `LOGOFF` to `AUTHENTICATION`
```

Comment:
I think `LOGOFF` should be in tags <>? Not sure about the others.

Reply:
<> the tags are to denote not a method but an action and logoff is a clear message that we control so should remain without tags (i think haha) ill ask the team

Reply:
Yes, that's right. This instance is correct, while other `<LOGON>`/`<LOGOFF>` elsewhere are wrong.